sdcm/cluster.py

```diff
@@ -2013,6 +2029,8 @@ def _parse_cs_summary(lines):
 
         for line in lines:
             line.strip()
+            if not line:
+                continue
```

Comment:
Why needed?

Reply:
Because if there is empty line in the output, parsing fails.

Reply:
It's the same as what written few lines below:
if line == '':
                 continue